### DISCUSSION TITLE
How to access HRRR smoke data from experimental version for Sep. 2020

### DISCUSSION THREAD

**Discussion by BBozorgmehr** [NOAA-GSL/GSL-Regional-Model-Forum#35] [Q&A] [answered]
Hello everyone,

I need to access the experimental smoke data for September 2020. I was wondering if anyone has any ideas of where it's been archived (if it is archived at all) and how I can access it. 

Your help is highly appreciated.

Best,
Behnam

**Comment by EricJames-NOAA** [comment]
@BBozorgmehr thanks for the question!  Yes, prior to Dec 2020 the smoke forecasts from HRRR were experimental.  There isn't yet a publicly available archive (although we are working on it), but we can provide data from our NOAA storage system.  

Can you share a bit more about which fields you are interested in (i.e., near-surface smoke or 3D smoke, or column-integrated smoke), and whether you need hourly forecasts, or just analyses?

**Reply by BBozorgmehr** [reply]
@EricJames-NOAA Thank you very much for your response, Eric! It would be great if the data become publicly available.

For now, we are mostly interested in near-surface smoke ("MASSDEN_8maboveground"). However, we might need 3D smoke or column-integrated smoke in the future. We just need the analyses for now.

I appreciate your help greatly.

Best,
Behnam

**Reply by BBozorgmehr** [reply]
Hi @EricJames-NOAA,

I was wondering if you can provide us with the smoke data from your storage system. We really appreciate your help.

Best,
Behnam

**Reply by EricJames-NOAA** [reply]
@BBozorgmehr yes I have the data ready!  I should be able to upload it today to our anonymous FTP server.  I will send you an email when it is ready.  Thanks for your patience.

Eric

**Reply by BBozorgmehr** [reply]
Hi @EricJames-NOAA,

Thank you very much for sending me the near-surface smoke data. It seems like we need more than the near-surface smoke for our downscaling. Is it possible for you to be kind enough to send me the 3D smoke, or column-integrated smoke for September 2020?
 
 I appreciate your help.
 
 Behnam

**Reply by EricJames-NOAA** [reply]
@BBozorgmehr Yes, we should be able to fairly easily provide the column-integrated smoke for Sep 2020.  I will work on getting the data ready, and then post on our server again.  It might be a week or two before it is ready.

Eric

**Reply by BBozorgmehr** [reply]
That would be great! Thank you very much!

Behnam

**Reply by msw17002** [reply]
Hello, @EricJames-NOAA.

I actually have a similar query.

May I access the same data from 2020-08-15 to 2020-10-17?

**Reply by EricJames-NOAA** [reply, marked as answer]
@BBozorgmehr I have uploaded the Sep 2020 data to our anonymous FTP server as before.

@msw17002 if you could go ahead and send me an email, I can send you access instructions for the FTP server.  You can find my email near the top of this document https://rapidrefresh.noaa.gov/hrrr/HRRRsmoke/HRRR-Smoke_VIIRS_Activefire_user_guide.pdf

**Reply by BBozorgmehr** [reply]
@EricJames-NOAA Thank you very much for providing this data for us. This will help us a lot to downscale the HRRR smoke data to our microscale diagnostic model. I downloaded the data and you can take it off of your FTP server.
